Docs/ProjectSpec.md

```diff
@@ -465,6 +467,33 @@ targets:
 
 The above will generate 2 targets named `MyFramework_iOS` and `MyFramework_tvOS`, with all the relevant platform build settings. They will both have a `PRODUCT_NAME` of `MyFramework`
 
+### Supported Platforms
+
+This will provide a mix of default build settings for the chosen platforms. It can be any of the following:
+
+- `iOS`
+- `tvOS`
+- `macOS`
+- `macCatalyst`
+
+Note that the definition of supported platforms will skip the `Multi Platform targets` generation.
+
+```yaml
+targets:
+  MyFramework:
+    sources: MyFramework
+    platform: iOS
```

Comment:
Would we even need to specify `platform` in this case? Having them both there is a little confusing

Reply:
Hi, I have created a default new project with Xcode 14 and it seems that you need a SDKROOT, a base SDK to use.

SupportedPlatform from the other end helps you to support multiple platform in the same target mixing together target families and other build settings.